libs/agency/feature/dashboard/src/lib/components/notes/note-form/note-form.component.ts

```diff
@@ -169,47 +193,58 @@ export class NoteFormComponent {
   }
 
   public saveNote() {
-    const formValue = this.noteFormGroup.value;
-    const noteModel = new NoteModel();
-
-    noteModel.body = formValue.body ?? '';
-    noteModel.title = formValue.title ?? '';
-    noteModel.type = formValue.type ?? '';
-
-    if (this.mode === NoteFormMode.Edit) {
-      // Edit existing note
-      this._transactionDetailService
-        .editNote$(this.transactionId, this._noteId, noteModel)
-        .pipe(
-          take(1),
-          tap(_ => {
-            this._nuverialSnackBarService.notifyApplicationSuccess();
-            this.navigateToNotes();
-          }),
-          catchError(_ => {
-            this._nuverialSnackBarService.notifyApplicationError();
-
-            return EMPTY;
-          }),
-        )
-        .subscribe();
+    this.formErrors = [];
+
+    if (this.noteFormGroup.valid) {
```

Comment:
let's refactor this to 
```suggestion
    if (!this.noteFormGroup.valid) {
      this.formErrors = HandleFormErrors(this.noteFormGroup, this.formConfigs);
      MarkAllControlsAsTouched(this.noteFormGroup);
      return;
    }
```

so we don't need to wrap the whole function in a if else